kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2212,6 +2212,22 @@ object KyuubiConf {
       .checkValue(_ >= 1000, "must >= 1s if set")
       .createOptional
 
+  val OPERATION_TIMEOUT_POOL_SIZE: ConfigEntry[Int] =
+    buildConf("kyuubi.operation.timeout.pool.size")
+      .doc("Number of threads in the per-OperationManager timeout scheduler used for " +
+        "operation-level timeout monitoring.")
```

Comment:
we should not expose OperationManager concept to user docs, for user, it's a global pool